apps/gro/steps.js

```diff
@@ -133,7 +133,94 @@ module.exports = {
     template: 'email',
     fields: ['email-text'],
     continueOnEdit: true,
-    next: '/post'
+    next: '/country'
+  },
+  '/country': {
+    fields: [
+      'country'
+    ],
+    forks: [{
+      target: '/address-outside',
+      condition: function checkIfNotUK(req) {
+        return req.form.values.country !== 'United Kingdom';
+      }
+    }],
+    continueOnEdit: true,
+    next: '/postcode'
+  },
+  '/postcode': {
+    template: 'postcode',
```

Comment:
you only have to specify the template name if it's different from the route. I.e. in this case, not 'postcode'